tools/loader/ImageClassifier.cpp

```diff
@@ -83,11 +83,18 @@ llvm::cl::opt<ImageNormalizationMode> imageMode(
 llvm::cl::alias imageModeA("i", llvm::cl::desc("Alias for -image_mode"),
                            llvm::cl::aliasopt(imageMode),
                            llvm::cl::cat(imageLoaderCat));
+
+
+llvm::cl::opt<std::string> loadimageFormat(
+    "image_format",
+    llvm::cl::desc("Specify which image format to use"));
+
 } // namespace
 
 /// Loads and normalizes all PNGs into a tensor in the NCHW 3x224x224 format.
 void loadImagesAndPreprocess(const llvm::cl::list<std::string> &filenames,
```

Comment:
Maybe a better technique would be to transpose the tensor outside of this method. It would simplify things.